src/calendar/__specs__/month_spec.tsx

```diff
@@ -16,7 +16,7 @@ describe('Month', () => {
     wrapper = shallow(<Month {...props} />)
     instance = wrapper.instance()
     mockEvent = {
-      currentTarget: { value: date },
+      currentTarget: { dataset: { simpleReactCalendarDay: date } },
```

Comment:
not sure, but currently `data-simple-react-calendar-day` where we are rely on some data.
Possbile `data-sc-date` or just `data-date`, WDYT?

Reply:
Peter’s variant looks better to me